modules/local/clair3/main.nf

```diff
@@ -1,5 +1,5 @@
 //
-// Two separate c3 processes for now, probably will combine later into 1
+// Two separate Clair3 processes for now, probably will combine later into 1
 //
 process CLAIR3_POOL {
```

Comment:
I think these should probably be one process? What's the big difference between them?